src/main/java/org/broadinstitute/hellbender/tools/copynumber/gcnv/GermlineCNVNamingConstants.java

```diff
@@ -0,0 +1,17 @@
+package org.broadinstitute.hellbender.tools.copynumber.gcnv;
+
+/**
+ * This class stores naming standards in the Germline CNV caller
+ */
+public final class GermlineCNVNamingConstants {
+
+    protected final static String COPY_NUMBER_POSTERIOR_FILE_NAME = "log_q_c_tc.tsv";
+
+    protected final static String COPY_NUMBER_STATE_STRING_START = "COPY_NUMBER_";
+
+    protected final static String INTERVAL_LIST_FILE_NAME = "interval_list.tsv";
+
+    protected final static String GC_COMPONENTS_FILE_NAME = "mu_z_sg.tsv";
```

Comment:
We can just add these unused constants back later if need be.

Reply:
`GC_CURVE_FILE_NAME`

Reply:
Done